src/mold.h

```diff
@@ -3097,11 +3097,11 @@ inline bool is_c_identifier(std::string_view s) {
   if (s.empty())
     return false;
 
-  auto is_alpha = [](char c) {
+  static constexpr auto is_alpha = [](char c) {
```

Comment:
I wouldn't add `constexpr` to a function unless it is necessary.

Reply:
Shall I change it to `static const` or drop it entirely?

Reply:
I'd drop all but `[&]` -> `[]` changes.

Reply:
I could have explained this better. The thing here is that making `is_alpha` static allows us to make `is_alnum` non-capturing. If we want that, then my instinct is to make `is_alpha` a (compile-time or run-time) constant.

If we drop the `static`, we have to re-add the `[&]` to `is_alnum`.

Reply:
It's not important, so let's keep it as-is. I believe the compiler is smart enough to optimize these lambdas out.

Reply:
Okay, done. Do you wish to keep the commit that converts the `for`-loop into a call to `std::any_of()` (following the example in `passes.cc:877`)?

Reply:
Not really. These are a matter of taste, and I'm honestly not interested in accepting changes like that. I might make a change like that to improve code based on my preferences, but I don't want contributors to make similar non-functional changes because everyone has a different taste.

In fact, since I personally care about details like that and fix them, you don't have to care about them. If you look closely at this project, you might notice that I just accept incoming pull requests without too much code review and then edit the changes after merging. I do this to not waste time on both sides.

Reply:
Dropped; sorry about misunderstanding you there.